.github/ISSUE_TEMPLATE.md

```diff
@@ -19,6 +25,22 @@ Paste your log here, between the backticks. It can be:
   - npm --verbose output,
   - or contents of npm-debug.log,
   - or output of node-gyp rebuild --verbose.
+Include the command you were trying to run.
+
+This should look like this:
+
+>npm --verbose
+npm info it worked if it ends with ok
+npm verb cli [
+npm verb cli   'C:\\...\\node\\13.9.0\\x64\\node.exe',
+npm verb cli   'C:\\...\\node\\13.9.0\\x64\\node_modules\\npm\\bin\\npm-cli.js',
+npm verb cli   '--verbose'
+npm verb cli ]
+npm info using npm@6.13.7
+npm info using node@v13.9.0
+
+Usage: npm <command>
+(...)
```

Comment:
Should we talk about three backticks on a line by themselves above the first line of the log and the same below the last line of the log?

Reply:
I hope that people will just overwrite the example output, not paying to much attention to the backticks.  We can add one more extra line here to make it easier for users to use backticks correctly.